packages/wrangler/src/publish.ts

```diff
@@ -236,8 +236,12 @@ export default async function publish(props: Props): Promise<void> {
     Or you could pass it in your terminal as \`--compatibility-date ${compatibilityDateStr}\`
 See https://developers.cloudflare.com/workers/platform/compatibility-dates for more information.`);
   }
-
   const triggers = props.triggers || config.triggers?.crons;
+
+  if (config.route === "") {
```

Comment:
what if one of config.routes is also ""?

Reply:
If we are allowing for empty strings for an explicit representation of "routes not being set" and `routes` is used for many routes is there a case where `routes = [ "", "some-route/* ]` or just `routes = [ "" ]` because I had originally coded this with those two scenarios in mind but it didn't fully make sense... like why would someone use `routes` for one route anyway?